src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -178,6 +178,11 @@ public List<ExceptionRenderingFormat> InnerFormats
             private set;
         }
 
+        /// <summary>
+        /// Get raw value, updates the sequence
+        /// </summary>
+        object IRawValue.GetRawValue(LogEventInfo logEvent) => logEvent.Exception;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename parameter 'logEvent' to 'logEventInfo' to match the interface declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS927)